rvv-intrinsic-rfc.md

```diff
@@ -566,6 +567,19 @@ Example:
 vint64m1_t vreinterpret_v_i32m1_i64m1(vint32m1_t src)
 ```
 
+### Reinterpret between vector boolean types and LMUL=1 (m1) vector integer types<a name="reinterpret-vbool-o-vintm1"></a>
+
+These utilities help users to convert vector boolean types to `LMUL=1` vector integer types, e.g. converting `vbool64_t` to `vint64m1_t`, `vbool16_t` to `vint8m1_t`. The reinterpret intrinsics only change the types of the underlying contents. It is a nop operation.
```

Comment:
What are the semantics of the extra bits that don't exist in the bool type when converting to the int type?

Reply:
The reinterpret function is a pseudo function that lets user goes over the type safe scheme boundary.

I think I was wrong in this paragraph.  Currently the v-spec specifies `v0` as the single supported mask register. The semantic of the reinterpret cast shall be a `vmv.v.v` between the mask register (`v0`) and the `m1` integer register (which can be anything in `v8` ~ v32`). Revised the paragraph.

Reply:
v0 is the only _input_ mask register. vmseq and similar can write any register with mask data.

The vbool64_t type only contains vlen/64 bits. Casting it to m1 leaves other (63/64)*vlen bits undefined. As far as the compiler is concerned we'd be allowed to have them be any value. It would be difficult for the user to control. I supposed a lmul_ext has the same issue so maybe this is expected?

Reply:
I was confused by myself in the previous comment, please disregard them.

I think the user should be aware of the mask register layout and know that the remaining bits are undefined.

Reply:
Added another note in [5bcf9a4](https://github.com/riscv-non-isa/rvv-intrinsic-doc/pull/221/commits/5bcf9a49f912c48ace1aea15cd10e2dec10af8a6) to aware users of the mask register layout.

Reply:
hm, is there some other way to go from vbool64_t to vbool1_t? Otherwise casting still seems important, even if some of the bits are undefined.

Reply:
That still possible by doing something like this: `vbool64_t` -> `m1` -> `vbool1_t`.

Reply:
Ah, perhaps I misunderstood, it sounded like there was a proposal not to support vbool64_t -> m1. I think that would be too limited.